src/compositemark/boxplot.ts

```diff
@@ -47,6 +47,8 @@ export const BOXPLOT_STYLES: BoxPlotStyle[] = ['boxWhisker', 'box', 'boxMid'];
 export interface BoxPlotConfig extends MarkConfig {
   /** Size of the box and mid tick of a box plot */
   size?: number;
+  /** Same as BoxPlotDef extent property */
```

Comment:
This is not an acceptable explanation that will show up in the documentation.  
```
    *  The default extent, which is used to determine where the whiskers extend to. The options are
    * - `"min-max": min and max are the lower and upper whiskers respectively.	    * - `"min-max": min and max are the lower and upper whiskers respectively.
-   * - `"number": A scalar (integer or floating point number) that will be multiplied by the IQR and the product will be added to the third quartile to get the upper whisker and subtracted from the first quartile to get the lower whisker.	+   * -  A scalar (integer or floating point number) that will be multiplied by the IQR and the product will be added to the third quartile to get the upper whisker and subtracted from the first quartile to get the lower whisker.
```


Also, if you push fixes, please do not squash and force push as I won't notice.

Reply:
So do you want what you posted instead of the following?
/** Same as BoxPlotDef extent property */

Reply:
Yes.   There is no "BoxPlotDef" appear in the documentation, so this wouldn't be understandable.

Reply:
I assume:
```"min-max": min and max are the lower and upper whiskers respectively.	```
showing up twice in what you typed above is a typo

Reply:
Yes, that's a typo. 

Not sure what to update on line 35.  I copy this from line 35 for you. 

The only exception is that I say "the default extent" here as this is a config. 
The term the default extent wouldn't apply in line 35.